doc/user/content/serve-results/metabase.md

```diff
@@ -26,12 +29,20 @@ Database name     | **materialize**
 Database username | Materialize user.
 Database password | App-specific password.
 
-If you require SSL/TLS encryption, you can configure the connection to use `sslmode=require`. For more details, check out the [Metabase documentation](https://www.metabase.com/docs/latest/administration-guide/databases/postgresql.html).
+If you require SSL/TLS encryption, you can configure the connection to use
+`sslmode=require`. For more details and troubleshooting, check the
+[Metabase documentation](https://www.metabase.com/docs/latest/administration-guide/databases/postgresql.html).
 
 ## Refresh rate
 
-By default, the lowest [refresh rate](https://www.metabase.com/docs/latest/users-guide/07-dashboards.html#auto-refresh) for Metabase dashboards is 1 minute. You can manually set this to a lower interval by adding `#refresh=1` (as an example, for a `1` second interval) to the end of the URL, and opening the modified URL in a new tab.
+By default, the lowest [refresh rate](https://www.metabase.com/docs/latest/users-guide/07-dashboards.html#auto-refresh)
+for Metabase dashboards is 1 minute. You can manually set this to a lower
+interval by adding `#refresh=1` (as an example, for a `1` second interval) to
+the end of the URL, and opening the modified URL in a new tab.
 
-Because Metabase queries are simply reading data out of self-updating views in Materialize, setting your dashboards to auto-refresh at lower rates will not have a significant impact on database performance.
+Because Metabase queries are simply reading data out of self-updating views in
+Materialize, setting your dashboards to auto-refresh at lower rates will not
+have a significant impact on database performance.
```

Comment:
I realize this was already here, but this is promising a bit too much from Materialize's performance. This should say something like "if you index appropriately, _then_ it will be okay"

Reply:
Changed to:

> Because Metabase queries are simply reading data out of self-updating views in
Materialize, setting your dashboards to auto-refresh at lower rates should not
have a significant impact on database performance. To minimize this impact, we
recommend carefully choosing an [indexing strategy](https://materialize.com/docs/sql/create-index/) for any objects serving results 
to Metabase.